src/components/Image/src/Image.vue

```diff
@@ -179,7 +186,14 @@ export default {
 		},
 
 		shouldGetImageDimensions() {
-			return this.shape !== 'square';
+			return this.shape !== 'square' || this.shape !== 'original';
+		},
+
+		transitionComponent() {
+			if (!this.disableTransition) {
+				return 'm-transition-fade-in';
+			}
+			return 'span';
```

Comment:
Might want to make this a `<div>` since `<img>` is the direct child to this component.

Reply:
I tried div and it messed up the image shape
![image](https://github.com/square/maker/assets/18740390/1ca6ebf4-a45e-45f2-95c0-8e9a255779d6)
![image](https://github.com/square/maker/assets/18740390/070d08f3-12af-4356-a405-fa877f42b510)